### PR TITLE
added product group and other contacts

### PR DESCRIPTION
**Pull request by ShellyXueHan**

### CODE REVIEW

**Comment by ShellyXueHan** [comment]
@mitovskaol I've added some description for `product_group` label. As it's a template not guidance, I won't be too wordy here. But I'll add some instruction on how to use it in the onboarding docs instead.

File: .github/ISSUE_TEMPLATE/annotation_update.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ What annotations/labels need to be changed? (Select all that apply)
 - [ ] Product Owner
 - [ ] Technical Steward
 - [ ] Security Contacts
+- [ ] Other Critical Contacts
+- [ ] Product Group (a label used to group together OpenShift Project Sets that serve as one product in general. This should be self-identified within your organization/ministry/branch/team)
```
Comment (mitovskaol) [inline .github/ISSUE_TEMPLATE/annotation_update.md:28]:
```suggestion
- [ ] Product Group (an optional label that can be used to group together OpenShift Project Sets that share a business function (e.g. Platform Services project sets are labeled as part of "Platform Services" product group.  The value for this label should be determined by the organization/ministry/branch/team that owns the Project Set)
```

File: .github/ISSUE_TEMPLATE/annotation_update.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ What annotations/labels need to be changed? (Select all that apply)
 - [ ] Product Owner
 - [ ] Technical Steward
 - [ ] Security Contacts
+- [ ] Other Critical Contacts
```
Comment (mitovskaol) [inline .github/ISSUE_TEMPLATE/annotation_update.md:27]:
```suggestion
- [ ] Other Critical Contacts (contact info for other stakeholders such as members of IMB, Architecture, Privacy, etc groups that are interested in receiving comms related to this project sets.
```

File: .github/ISSUE_TEMPLATE/openshift_project_request.md [hunk 3]
```diff
@@ -45,6 +45,7 @@ Fill out the following fields
 * Product Short Name: 
 * Product Full Name: 
 * Product Description: 
+* Product Group Name (if exists, used to group together OpenShift Project Sets that serve as one product in general):
```
Comment (mitovskaol) [inline .github/ISSUE_TEMPLATE/openshift_project_request.md:48]:
```suggestion
* Product Group Name (if exists, used to group together OpenShift Project Sets that are related to each other): 
```

File: .github/ISSUE_TEMPLATE/openshift_project_request.md [hunk 4]
```diff
@@ -64,7 +65,9 @@ For security purpose, please provide the following contact information to the em
   - Organization Code:
   - Organization Unit Code: 
   - (refer to https://dir.gov.bc.ca/)
-
+* Other Critical Contacts
+  - (this could be a MISO, Architecture team member, Privacy Specialist, etc)
```
Comment (mitovskaol) [inline .github/ISSUE_TEMPLATE/openshift_project_request.md:69]:
```suggestion
  - (this could be a member of IMB, Architecture, Privacy, etc groups)
```
